lms/djangoapps/monitoring/tests/test_monitoring_utils.py

```diff
@@ -0,0 +1,93 @@
+"""
+Tests for the LMS monitoring middleware
+
+Note: File named test_monitoring_utils.py instead of test_utils.py because of an error
+in Jenkins with test collection and a conflict with another test_utils.py file.
```

Comment:
Reviewer Note: Original Jenkins failure for `lms.lib.tests.test_utils`:
**Error Message**
```
collection failure
```
**Stacktrace**
```
import file mismatch:
imported module 'test_utils' has this __file__ attribute:
  /home/jenkins/edx-platform/lms/djangoapps/monitoring/tests/test_utils.py
which is not the same as the test file we want to collect:
  /home/jenkins/edx-platform/lms/lib/tests/test_utils.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```